index.html

```diff
@@ -857,111 +872,94 @@ <h2>
+        the behavior of other actors in the ecosystem, improvements in other
+        layers of the stack, new threats to user privacy, as well as changing
+        societal norms and regulations.
+      </p>
+      <p>
+        It is expected that the various groups involved in the design and implementation
+        of the Digital Credentials API actively monitor the evolving privacy
+        landscape and participate in the corresponding evolution of the API.
+      </p>
+      <section data-cite="vc-data-model#spectrum-of-privacy">
         <h3>
-          Data Retention
+          Spectrum of Privacy
         </h3>
-        <p class="issue" title="Work in progress">
-          Explain how verifiers might retain data and what the ecosystem does
-          to mitigate excessive data retention policies.
+        <p>
+          The Digital Credentials API serves a variety of use cases with different
+          grades of data disclosure and individual users with different preferences
+          depending on the context that they are in. Notably, the privacy properties
+          of a credential exchange mediated by this API could be mandated by
+          the legal and regulatory environment of an individual user.
         </p>
-      </section>
-      <section>
-        <h3>
-          Compliance with Privacy Regulations
-        </h3>
-        <p class="issue" title="Work in progress">
-          Explain to what extent the API complies with known privacy
-          regulations (e.g., consent) and what parts of those regulations are
-          not possible to enforce via the API (e.g., retention).
+        <p>
+          This means that some users may not want, or be allowed,
+          to use the most privacy-preserving means of exchanging credential information.
+          Nonetheless, user agents need to serve users with an experience
+          that is private by default and protect them from harm.
         </p>
-      </section>
-      <section>
-        <h3>
-          Selective and Unlinkable Disclosure
-        </h3>
-        <p class="issue" title="Work in progress">
-          Explain how selective disclosure and unlinkable disclosure help
-          preserve privacy as well as their limitations in doing so.
+        <p>
+          Because of this spectrum of preferences and use cases, it may be difficult for a user
+          agent to discern whether a user means to expose their personal information
+          or is being tricked into doing so. It is thus the user agent's responsibility
+          to ensure that every user understands what data they are sharing and who
+          will participate in the exchange of information, before the exchange
+          begins. It is further the user agent's responsibility to ensure that
+          all parties have a legitimate reason for facilitating the exchange.
         </p>
       </section>
       <section>
         <h3>
-          Phoning Home
```

Comment:
This is a [critical privacy property of digital ID](https://nophonehome.com/). It's relevant for the web API in terms of how/when wallets are notified about a request, evaluation of the request and issuance protocols to see whether presentation will enable phone-home, and whether revocation properties are included in such a way that might lead to phone home.

Reply:
That is totally fair and correct, but it seems to me that this is a case for #247 then, and doesn't need it's own separate section?

Reply:
As I noted, phoning home is not just a risk from wallets, and user choice over wallets (which is extremely unlikely) doesn't seem like a good solution anyway. I would rather not blame the user for not having done extensive due diligence over their wallet, even if they actually had free choice over it, which we know they will not.